sdk/python/foundation-models/system/finetune/Llama-notebooks/distillation/distillation_qa_math.ipynb

```diff
@@ -0,0 +1,700 @@
+    "    validation_file_path: Input = None,\n",
+    "):\n",
+    "    oss_distillation = distillation_pipeline_component(\n",
+    "        teacher_model_endpoint_name=teacher_model_endpoint_name,\n",
+    "        enable_chain_of_thought=enable_chain_of_thought,\n",
+    "        train_file_path=train_file_path,\n",
+    "        validation_file_path=validation_file_path,\n",
+    "        data_generation_task_type=\"NLU_QA\",\n",
+    "        # Finetune\n",
+    "        mlflow_model_path=input_finetune_model,\n",
+    "        model_asset_id=student_model.id,\n",
+    "        system_properties=system_properties,\n",
+    "        ## hyperparams\n",
+    "        learning_rate=0.00002,\n",
+    "        per_device_train_batch_size=1,\n",
+    "        num_train_epochs=3,\n",
+    "    )\n",
+    "\n",
+    "    return {\"output_model\": oss_distillation.outputs.output_model}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1724712286770
+    }
+   },
+   "outputs": [],
+   "source": [
+    "system_properties = {\n",
+    "    \"finetune_oss\": \"True\",\n",
+    "    \"model_asset_id\": student_model.id,\n",
+    "    \"PipelineType\": \"Finetune\",\n",
+    "    \"azureml.PipelineType\": \"Finetune\",\n",
+    "    \"azureml.ModelName\": student_model.name,\n",
+    "    \"azureml.original_model_id\": student_model.id,\n",
+    "    \"azureml.trainingData.assetId\": train_data_asset_id,\n",
+    "}\n",
+    "\n",
+    "json_str = json.dumps(system_properties).replace(\" \", \"\")\n",
+    "\n",
+    "system_properties_b64_encoded = base64.b64encode(json_str.encode(\"utf-8\")).decode(\n",
+    "    \"utf-8\"\n",
+    ")\n",
+    "print(f\"System properties => {system_properties_b64_encoded}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1724712291790
+    }
+   },
+   "outputs": [],
+   "source": [
+    "train_file_path_input = Input(type=\"uri_file\", path=train_data.path)\n",
+    "validation_file_path_input = Input(type=\"uri_file\", path=valid_data.path)\n",
+    "input_finetune_model = Input(type=\"mlflow_model\", path=student_model.id)\n",
+    "experiment_name = f\"distillation-{TEACHER_MODEL_NAME}\".replace(\".\", \"-\")\n",
+    "\n",
+    "finetuning_job = distillation_pipeline(\n",
+    "    teacher_model_endpoint_name=TEACHER_MODEL_ENDPOINT_NAME,\n",
+    "    enable_chain_of_thought=ENABLE_CHAIN_OF_THOUGHT,\n",
+    "    system_properties=system_properties_b64_encoded,\n",
+    "    input_finetune_model=input_finetune_model,\n",
+    "    train_file_path=train_file_path_input,\n",
+    "    validation_file_path=validation_file_path_input,\n",
+    ")\n",
+    "\n",
+    "finetuning_job.properties.update(system_properties)\n",
+    "print(f\"job property: {finetuning_job.properties}\")\n",
+    "\n",
+    "# pipeline_job.identity = UserIdentityConfiguration()\n",
```

Comment:
Should we remove this line instead of commenting it out?

Reply:
Good idea! I don't see any references to pipeline_job anywhere else, will remove them